### ISSUE TITLE
GTK Theme issues with Mate on Mint 12

### ISSUE THREAD

**Issue by JDougherty** [perberos/Mate-Desktop-Environment#63]
After a fresh install of Linux Mint 12 from the DVD, themes are not working properly in Mate. The specific error is:

"This theme will not look as intended because the required GTK+ theme engine 'murrine' is not installed"

I get the same error for different themes / theme engines. 

Per glebihan's help in the Linux Mint Help IRC, I followed the instructions at: http://blog.linuxmint.com/?p=1901

Tried restarting the computer afterwards but the issue persists. We verified that the Linux Mint romeo repository source is enabled and that the murrine-themes and gtk2-engines-murrine packages are installed. 

The system is a Lenovo Thinkpad T61p with Linux Mint 12 64-bit.

**Comment by perberos** [comment]
version?

``` sh
$ mate-about --version
```

**Comment by perberos** [comment]
#50

**Comment by pedroteixeira** [comment]
same problem here, are there any ways to fix it?

**Comment by JDougherty** [comment]
Sorry, I reverted to Mint 11 so I can't give any more information. Hopefully pedroteixeira can help.

**Comment by perberos** [comment]
@JDougherty no problem

@pedroteixeira update to version 2011.12.12

**Comment by Moshu** [comment]
Hello,

I have the same problem on a Sony Vaio VPCEH, Linux Mint 12 64-bit.

I have entered the version command and it is:

MATE Desktop Environment 2011.11.03

I see your advice is to update to version 2011.12.12 but the question is how could I perform this. In Synaptic I have all the packages at the latest version. Should I add a different repository for MATE?

Many thanks in advance and keep up the good MATE work, it's great to be able to use Gnome the old way.

-- edit: I have already added the Romeo unstable repository but after refresh it doesn't show any updates available. I also have gtk2-engines-murrine.
